### PR TITLE
Hystrix admin port pr1

### PR DESCRIPTION
**Pull request by trabetti**
some code review fixes + some original changes that "disappeared" from last commit

### CODE REVIEW

File: source/server/http/admin.cc [hunk 1]
```diff
@@ -663,7 +663,7 @@ AdminImpl::NullRouteConfigProvider::NullRouteConfigProvider()
 AdminImpl::AdminImpl(const std::string& access_log_path, const std::string& profile_path,
                      const std::string& address_out_path,
                      Network::Address::InstanceConstSharedPtr address, Server::Instance& server,
-                     Stats::Scope& listener_scope)
+                     Stats::ScopePtr&& listener_scope)
```
Comment (eliranroffe) [inline source/server/http/admin.cc:666]: Did you changed that?

File: source/server/http/admin.cc [hunk 2]
```diff
@@ -698,8 +698,7 @@ AdminImpl::AdminImpl(const std::string& access_log_path, const std::string& prof
           {"/runtime", "print runtime values", MAKE_ADMIN_HANDLER(handlerRuntime), false, false},
           {"/hystrix_event_stream", "send hystrix event stream",
            MAKE_ADMIN_HANDLER(handlerHystrixEventStream), false, false}},
-      listener_stats_(
-          Http::ConnectionManagerImpl::generateListenerStats("http.admin.", listener_scope)) {
```
Comment (eliranroffe) [inline source/server/http/admin.cc:702]: We changed that?

**Review by eliranroffe — APPROVED** [review]

File: source/server/http/admin.cc (hunk 1 above)
Comment (trabetti) [reply source/server/http/admin.cc:666]: no, it is their change that in the big merge was lost. one of the reasons the previous PR did not compile.

File: source/server/http/admin.cc (hunk 2 above)
Comment (trabetti) [reply source/server/http/admin.cc:702]: no, it is their change that in the merge was lost. one of the reasons the previous PR did not compile.

**Pull request merged**